src/stru_multi/4C_stru_multi_microstatic.cpp

```diff
@@ -717,10 +726,42 @@ void MultiScale::MicroStatic::output(
   }
 }  // MultiScale::MicroStatic::output()
 
+/*----------------------------------------------------------------------*
+ |  write runtime output (public)
+ *----------------------------------------------------------------------*/
+void MultiScale::MicroStatic::runtime_output(
+    const std::pair<double, int>& output_time_and_step, const std::string& section_name) const
+{
+  std::vector<double> out_cmat;
+  out_cmat.reserve(36);
+
+  for (int i = 0; i < 6; i++)
```

Comment:
This double for loop including the `reserve` statement before can be condensed to `std::vector<double> out_cmat(macro_cmat_.values(), macro_cmat_.values()+36);` if I do not miss something.